OWML.ModLoader/Owo.cs

```diff
@@ -51,28 +52,36 @@ public void LoadMods()
             var priorityMods = _modFinder.GetMods().Where(mod => mod.Manifest.PriorityLoad).ToList();
             var sortedPriority = _sorter.SortMods(priorityMods);
 
-            var modNames = _modFinder.GetMods().Where(mod => mod.Config.Enabled).Select(mod => mod.Manifest.UniqueName).ToList();
+            var modNames = _modFinder.GetMods().Where(mod => mod.Config.Enabled)
+                .Select(mod => mod.Manifest.UniqueName).ToList();
             var sortedMods = sortedPriority.Concat(sortedNormal);
 
-            foreach (var mod in sortedMods)
+            foreach (var modData in sortedMods)
             {
-                foreach (var dependency in mod.Manifest.Dependencies)
+                var isMissingDependency = false;
+                if (modData.Config.Enabled)
                 {
-                    if (!modNames.Contains(dependency) && mod.Config.Enabled)
+                    var missingDependencies = modData.Manifest.Dependencies
+                        .Where(dependency => !modNames.Contains(dependency));
```

Comment:
if dependencies doesn't contain anything (can't remember if it would be null in that case?) would this break?

Reply:
it would be empty I think. I didn't get any errors while testing with CustomLoader enabled

Reply:
yeah, dependencies defaults to an empty string array, so this should be good 👍

Reply:
It defaults to empty list: 
![image](https://user-images.githubusercontent.com/3322367/87411016-5dbc3280-c5c7-11ea-8818-c711703d2a80.png)